### PR TITLE
Issue 399 visuals fixed

### PR DESCRIPTION
**Pull request by FreeLikeGNU**
Resolved visual problems found in issue https://github.com/bafolts/terraforming-mars/issues/399
Screenshot_2020-03-16_11-51-16
Screenshot_2020-03-16_11-49-16

### CODE REVIEW

File: src/HTML_data.ts [hunk 1]
```diff
@@ -5136,7 +5136,7 @@ export const HTML_DATA: Map<string, string> =
 ["EcoLine",`
     <div class="tag tag1 tag-plant"></div>
     <div class="corporationLabel">CORPORATION</div>
-    <div class="contentCorporation">
+    <div class="contentCorporation" style="color:#020202;">
```
Comment (bafolts) [inline src/HTML_data.ts]: This doesn't seem to have been added to every `contentCorporation`. I would create a separate class for this style color and use it where it is being applied. Which corporations were picked to use this color if not all of them?

Comment (bafolts) [reply src/HTML_data.ts]: This all looks fine to me once a CSS class is utilized for the `style="color:#020202;"`. I am not sure which corporations are getting this color but then the overall div class can be similar to `class="contentCorporation reasonForColor"`.

Comment (FreeLikeGNU) [reply src/HTML_data.ts]: I've made the requested changes and fixed a few more things as well

**Comment by alrusdi** [comment]
I've already fixed resources count visibility problem. Please remove changes from HTML_data - it will be a huge pain in the future. We should rework that module...

File: src/HTML_data.ts [hunk 2]
```diff
@@ -5518,7 +5518,7 @@ export const HTML_DATA: Map<string, string> =
           on this card.)
           </div>
         </div>
-        <div  class="celestic" style="font-size:24px;box-shadow:6px 6px 6px grey;margin-left: 9px;
+        <div  class="celestic" font-size:24px;box-shadow:6px 6px 6px grey;margin-left: 9px;
```
Comment (bafolts) [inline src/HTML_data.ts]: Does this work to not include the `style` attribute. This HTML was copied from the ssimoneoff cards repository and should be using Vue.js templates. Eventually each of these should move to their own template file. @alrusdi has done most of the CSS work up until this point so he is probably best to approve or reject these changes. I would like to see `HTML_data` eventually go away though and move towards Vue.js templates for each of these cards.

Comment (FreeLikeGNU) [reply src/HTML_data.ts]: I've reverted to upstream HTML_data.ts however the "points" need to be moved in a few corporation cards such as Arklight and Pristar. Maybe resolved in another PR?
Screenshot_2020-03-17_11-47-26

Comment (alrusdi) [reply src/HTML_data.ts]: Please do it by css, add class "card-arklight" and "card-pristar" in HTML_data.ts, then fix fontt size or positioning in css rules.

Comment (FreeLikeGNU) [reply src/HTML_data.ts]: I've removed my changes to HTML_data.ts while a better way to fix the point boxes in corporations is under discussion.

**Pull request merged**